### PR TITLE
Fix SA1615/SA1623 warnings

### PR DESCRIPTION
**Pull request by iamdmitrij**
# Pull Request

## The issue or feature being addressed
#1290

## Details on the issue fix or feature implementation

- [x] Fix [SA1615](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1615.md) by adding documentation to return values 
- [x] Fix [SA1623](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1623.md) by prefixing property docs to `Gets`/`Sets`

## Confirm the following

- [x]  I started this PR by branching from the head of the default branch
- [x]  I have targeted the PR to merge into the default branch
- [ ]  I have included unit tests for the issue/feature
- [x]  I have successfully run a local build

### CODE REVIEW

**Comment by iamdmitrij** [comment]
Noticed that [SA1623](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1623.md) can be fixed without suppressing rule. Will push new changes.

File: src/Polly/PolicyBuilder.cs [hunk 1]
```diff
@@ -87,12 +87,12 @@ internal PolicyBuilder(ExceptionPredicates exceptionPredicates)
...
 
     /// <summary>
-    /// Predicates specifying exceptions that the policy is being configured to handle.
+    /// Gets the predicates specifying exceptions that the policy is being configured to handle.
     /// </summary>
     internal ExceptionPredicates ExceptionPredicates { get; }
 
     /// <summary>
-    /// Predicates specifying results that the policy is being configured to handle.
+    /// Gets the Predicates specifying results that the policy is being configured to handle.
```
Comment (iamdmitrij) [inline src/Polly/PolicyBuilder.cs:95]: Spelling 
```suggestion
    /// Gets the predicates specifying results that the policy is being configured to handle.
```

**Comment by iamdmitrij** [comment]
[SA1623](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1623.md) is now fixed without `.editorconfig` suppression.

File: src/Polly/Context.cs [hunk 2]
```diff
@@ -25,24 +25,25 @@ public Context()
...
-    /// A key unique to the call site of the current execution.
+    /// Gets a key unique to the call site of the current execution.
     /// <remarks>Policy instances are commonly reused across multiple call sites.  Set an OperationKey so that logging and metrics can distinguish usages of policy instances at different call sites.</remarks>
     /// <remarks>The value is set by using the <see cref="Context(string)"/> constructor taking an operationKey parameter.</remarks>
     /// </summary>
     public string OperationKey { get; }
 
     /// <summary>
-    /// A Guid guaranteed to be unique to each execution.
+    /// Gets a Guid guaranteed to be unique to each execution.
```
Comment (martincostello) [inline src/Polly/Context.cs:46]:
```suggestion
    /// Gets a GUID guaranteed to be unique to each execution.
```

**Review by martincostello — APPROVED** [review]

**Comment by martincostello** [comment]
Thanks again!

**Pull request merged**